docs/updates/migrate-v3.md

```diff
@@ -171,7 +171,7 @@ The following processors have been removed:
 
 | UI5 Tooling v2              | UI5 Tooling v3              | Note |
 | --------------------------- | --------------------------- | ------------------------- |
-| createDebugFiles<br/>uglify | minify                      | The minify task is executed earlier, before the bundling process takes place. Any existing 'beforeTask' or 'afterTask' configuration of custom tasks might need to be adapted to cater for this change. |
+| createDebugFiles<br/>uglify | minify                      | The minify task is executed earlier, before the bundling process takes place. Any existing `beforeTask` or `afterTask` configuration of custom tasks might need to be adapted to cater for this change. <br>To adapt, you can use the `generateResourcesJson` for subscription before or after the last standard task. By default, `generateResourcesJson` is disabled, but you can still subscribe to it, ensuring your custom tasks execute in the correct order. |
```

Comment:
> you can use the `generateResourcesJson` for subscription

isn't something missing here?
I would either expect "you can use the `generateResourcesJson` task for subscription" or "you can use `generateResourcesJson` for subscription"

@KlattG

Reply:
Done